plugins/orchestrator-backend/src/types/apiResponse.ts

```diff
@@ -0,0 +1,31 @@
+export interface ApiResponse {
```

Comment:
We may review this later, but it should follow a pattern in all the orchestrator backend APIs, for example, here the return payload contains(result, message), but this is not what other APIs follow, at some point we need to standardize all APIs, this is probably related to the task Gloria is working on with the OpenApi for the backend plugin.

Reply:
sure, noted.